strings/phobia.json

```diff
@@ -108,6 +108,16 @@
 		"viper",
 		"python"
 	],
+	"birds": [
+		"owl",
+		"griffon",
+		"poly",
+		"kiki",
+		"yaya",
+		"vox",
```

Comment:
Not relevant for /tg/. I believe it would be more suited on [ParadiseSS13/Paradise](https://github.com/ParadiseSS13/Paradise) instead.

Reply:
Not relevant for /tg/. I believe it would be better suited on [ParadiseSS13/Paradise](https://github.com/ParadiseSS13/Paradise) instead.